src/modules/common/App.jsx

```diff
@@ -4,50 +4,56 @@ import {ConnectedRouter} from 'react-router-redux';
 import styled from 'styled-components';
 
 import HomeContainer from '../home/HomeContainer';
+import HealthContainer from '../health/HealthContainer';
+import RegisterContainer from '../register/RegisterContainer';
 
 // Import common styles
 import './styles/base.scss';
 
 export default function App({history}) {
-
-  const Nav = styled.nav`
```

Comment:
I didn't change this manually.. this was from Prettier styling based on our .prettierrc config. See my overall comment about styling; I'll make another PR which auto-formats the entire codebase after this is merged.